src/Dumplie/Customer/Application/Query/Result/Cart.php

```diff
@@ -60,6 +83,6 @@ public function totalPrice() : float
      */
     public function currency() : string
     {
-        return $this->isEmpty() ? null : $this->items[0]->currency();
+        return $this->isEmpty() ? '' : $this->items[0]->currency();
```

Comment:
null was better, there can't be a cart without currency, error is better than empty string